locale/pt-br/index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: index.hbs
+labels:
+  banner: Liberaçao de versões seguras importantes, por favor atualize agora!
+  current-version: Versão Atual
+  download: Baixar
+  download-for: Baixar para
+  other-downloads: Outros Downloads
+  other-lts-downloads: Outros Downloads LTS
+  other-current-downloads: Outros Downloads Atuais
+  current: Atual
+  lts: LTS
+  tagline-current: Últimas Features
+  tagline-lts: Recomendado Para Maioria dos Usuários
+  changelog: Changelog
+  api: Documentação da API
+  version-schedule-prompt: Ou verifique a
+  version-schedule-prompt-link-text: Cronograma do LTS.
+  newsletter: true
+  newsletter-prefix: Inscreva-se para
+  newsletter-postfix: ", as Atualizações Mensais Oficial do Node.js."
+---
+
+Node.js® é um runtime JavaScript desenvolvido com o [motor JavaScript V8 do Chrome](https://developers.google.com/v8/).
```

Comment:
I don't know if it's a good idea to translate `Chrome's V8 JavaScript engine` to `motor JavaScript V8 do Chrome` I think it is not usual to translate it this way

Reply:
I think the same thing as well.. changing here! Thanks @Drieger